terminal/terminal.go

```diff
@@ -41,6 +41,10 @@ func New(client service.Client, conf *config.Config) *Term {
 	}
 }
 
+func (t *Term) Close() {
+	t.line.Close()
+}
+
 // Run begins running dlv in the terminal.
 func (t *Term) Run() (int, error) {
 	defer t.line.Close()
```

Comment:
nit: let's call `t.Close()` here as well to stay consistent with how we cleanup Term.